store.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/checkbox-radio.css" >
+    <title>Karma form</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navigation">
+            <!-- Hamburger menu -->
+            <div class="mobile-container">
+                <img class="logo-svg" src="img/karma-logo.svg">
+                <div class="topnav">
+
+                    <div id="myLinks">
+                        <strong><a class="nav-li" href="#">Meet Karma</a></strong>
+                        <a class="nav-li" href="#">How is work</a>
+                        <a class="nav-li" href="#">Store</a>
+                        <a class="nav-li" href="#">Blog</a>
+                        <a class="nav-li" href="#">Help</a>
+                        <a class="nav-li" href="#">Login</a>
+                    </div>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```

Comment:
This does work on index.hlml, but you don't have myFunction defined in store.hlml.

Let's make it work on both pages. Some considerations:
 - pick a good name for the function. `myFunction` is not very descriptive and I need to look at the implementation to understand what it does. Would `expandCollapsNavigation` work better?
 - Let's put the implementation in separate js file that you can load on both web pages. Check out [script element examples](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#examples).